test/helpers.py

```diff
@@ -145,10 +144,9 @@ def check_updatable_packages(self, specifications_only=True):
                 current = min(inst_vs, key=CondaPackageHelper.semantic_cmp)
                 newest = avail_vs[-1]
                 if avail_vs and current != newest:
-                    if (
-                        CondaPackageHelper.semantic_cmp(current) <
-                        CondaPackageHelper.semantic_cmp(newest)
-                    ):
+                    if CondaPackageHelper.semantic_cmp(
+                        current
+                    ) < CondaPackageHelper.semantic_cmp(newest):
```

Comment:
This definitely was better.

Reply:
This was better before? I agree. I don't see a way to force black into better formatting in this case though. Perhaps by adding another `and` ? Hmmm I'll try it.

Reply:
Yes, it was better before black changes :(
Thanks 👍